mep/people/templates/people/member_list.html

```diff
@@ -20,6 +20,23 @@
         <label class="query">
             {% render_field form.query aria-label=form.query.label %}
         </label>
+        <fieldset>
+                <legend>Filter by:</legend>
+                <div class="toggle-filters">
+                    {% render_field form.has_card aria-label=form.has_card.label class+="sr-only sr-only-focusable" %}
+                    <label for="id_has_card">
+                        {# Wrapping attribute in <span> for CSS targeting #}
+                        <span>Card <span class="count">{{ members.get_facets.has_card_b.true }}</span></span>
+                        <img class="filter icon" src="{% static 'img/icons/member_card.svg' %}" alt="card icon">
+                        <div class="info-img tooltip" role="img" aria-label="This filter will narrow results to show only members whose library records are available." tabindex=0>
```

Comment:
these classes are starting to look sorta BEM-y, which isn't consistent with the rest of the project. in general I tend to go in the opposite direction of BEM: try writing a semantic selector without classes/IDs at all, and if that's not sufficient add only the classes necessary to disambiguate.